tooling/nargo_fmt/src/formatter/attribute.rs

```diff
@@ -136,6 +140,29 @@ impl<'a> Formatter<'a> {
         self.write_right_bracket(); // ]
     }
 
+    fn format_fuzz_attribute(&mut self, fuzz_scope: FuzzingScope) {
+        self.write_current_token_and_bump(); // #[
+        self.skip_comments_and_whitespace();
+        self.write_current_token_and_bump(); // test
```

Comment:
```suggestion
        self.write_current_token_and_bump(); // fuzz
```

though if they `fuzz` attribute is going to be removed then it doesn't matter.

Reply:
Thanks